ui/src/services/loki/api.js

```diff
@@ -0,0 +1,82 @@
+//@flow
+import ApiClient from '../ApiClient';
+
+let lokiApiClient: ?ApiClient = null;
+
+export function initialize(apiUrl: string) {
+  lokiApiClient = new ApiClient({ apiUrl });
+}
+
+type MetrixValue = {
+  metric: {
+    [labelName: string]: string,
+  },
+  values: [number, string],
+}[];
+type StreamValue = {
+  stream: {
+    [labelName: string]: string,
+  },
+  values: [string, string],
+}[];
+type LokiQueryResult = {
+  status: 'success',
+  data: {
+    resultType: 'matrix' | 'streams',
+    result: MetrixValue | StreamValue,
+    // Those statistics allow users to understand the amount of data processed and at which speed.
+    // https://grafana.com/docs/loki/latest/api/#Statistics
+    stats: {},
+  },
+};
+
+// by default to get the last 7day alerts from Loki
+export function getLast7DaysAlerts(): LokiQueryResult {
+  if (!lokiApiClient) {
+    throw new Error('lokiApiClient should be defined');
+  }
+
+  const query = "{container='alertmanager-logger'}";
```

Comment:
after discussing with @gdemonet we should have kept the namespace (it is for retrieving the log from a specific loki pod and has nothing to do with namespacing in alert manager which confused us)

Reply:
OK. Thanks for the update!!